### ISSUE TITLE
[Fable accessibility issue] Cannot read content in 400% when using a screen magnifier

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#4607]
### Steps to recreate
Go to https://catalog.princeton.edu/catalog/99131272367806421
Zoom in 400%
The harmful content feedback bar takes 1/3 of the page making it difficult for the user to read the content.

### URL
Example record page: https://catalog.princeton.edu/catalog/99131272367806421

### Issues
User cannot read the page content without a ridiculous amount of scrolling 

### Severity
High

### Recommended fix
Remove css sticky attribute

This issue came up in one of the Fable user testing sessions with a screen magnifier tool. 
@sandbergja @beth-german @Ka-YT 

This is already [fixed in the catalog](https://github.com/pulibrary/orangelight/pull/4605). We're creating this ticket to document the issue.

**Comment by christinach** [comment]
resolved by https://github.com/pulibrary/orangelight/pull/4605
